source/stylesheets/corporate/sections/_team.sass

```diff
@@ -4,8 +4,12 @@
 
 #team
   padding-bottom: $sectionsMargin
-  .col:hover .avatar
-    @include filter(none)
+  .col
+    &:hover .avatar
+      @include filter(none)
+    &:nth-child(4)
+      margin-left: 0
+      margin-right: 0
```

Comment:
I'm not really OK with styles in the `&:nth-child(4)` selector.

Without these style changes:
![alpinelab_website](https://cloud.githubusercontent.com/assets/61271/9134195/76386996-3cf1-11e5-9fd3-2269ee12bcb9.png)

With these style changes (9e709327):
![alpinelab_website-styles_change](https://cloud.githubusercontent.com/assets/61271/9134204/99b5c67a-3cf1-11e5-8233-e850a4004ac7.png)